.github/workflows/pytest.yaml

```
@@ -0,0 +1,28 @@
+name: pytest
+
+on:
+  push:
+    branches: [main, develop, 73-initialize-pytest]
+  pull_request:
+    branches: [main, develop, 73-initialize-pytest]
+
```

we can remove your branch now
```suggestion
    branches: [main, develop]
  pull_request:
    branches: [main, develop]

```

Since we have the `pull_request` here, you don't need to explicitely have your branch name in here, you can just open a Draft PR in github and it'll run the actions for you

---

whoops, I keep forgetting this. Thanks Pierre!